crates/cxx-qt-gen/test_inputs/signals.rs

```diff
@@ -6,20 +6,33 @@ mod ffi {
         type QPoint = cxx_qt_lib::QPoint;
     }
 
+    unsafe extern "C++Qt" {
+        include!(<QtCore/QTimer>);
+        /// QTimer
+        type QTimer;
+
+        /// When the QTimer timeout occurs
+        #[qsignal]
+        pub(self) fn timeout(self: Pin<&mut QTimer>);
+    }
+
     unsafe extern "RustQt" {
         #[qobject]
         type MyObject = super::MyObjectRust;
 
         #[qsignal]
         fn ready(self: Pin<&mut MyObject>);
 
+        #[qsignal]
+        fn connected(self: Pin<&mut MyObject>, url: &QUrl);
```

Comment:
I think this test can go, the `data_changed` signal already includes a by-reference parameter.